src/components/Cell/Cell.scss

```diff
@@ -3,8 +3,8 @@
     border-radius: 5px;
     border: solid white 3px;
     margin: 10px;
-    height: 2.8em;
-    width: 2.8em;
+    height: 2.5em;
```

Comment:
unfortunately, the cells still overflow out of the grid on my Mac screen

Reply:
actually just setting some min heights for the time being